webdriverio/webdriver/examples/visual-e2e/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "examples",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test.visual.sauce": "npx wdio test/configs/wdio.shared.conf.js",
```

Comment:
What script is doing what? The `test.visual.sauce` is referring to the shared one, the `test.visual.crossplatform` to the cross platform.

My advice would be (see also the best practices folder):
- have a shared one that only contains framework setup, not caps, no Sauce reference
- have a sauce shared with only the sauce config extending the shared one
- have separate config files that will be triggered with the scripts here and only extend the sauce.shared with different caps

Reply:
@wswebcreation please help, we need one simple test that anyone can get running by looking at one file and one config, a quickstart test. I think a user shouldn't be parsing through multiple configs to understand how to run a simple test. This is how users get started.
Next, the most common and problematic use case is users that want to run cross-browser/cross-platform, and that looks like this https://github.com/nadvolod/react-projects/blob/923cb86c094f0431b8fbad7773946abfda45c7b0/01-birthday-reminder/setup/wdio.conf.js#L28-L42 .

Reply:
@wswebcreation I created a new issue from your request here. It's the only issue remaining and blocking this code from being public. Since the current production code is not working and outdated, I will merge this PR and circle back to resolving the issue #157